.github/actions/release-image/action.yml

```diff
@@ -137,8 +137,9 @@ runs:
 
     # Build and push
     - id: apko
-      uses: chainguard-images/actions/apko-snapshot@main
+      uses: mattmoor/actions-2/apko-snapshot@dde1a99a719e367df7253ce1b57e5520b661cdfb
       with:
+        apko-image: ghcr.io/wolfi-dev/apko:latest@sha256:3f7077f5a9d74747c03c03e18d202e81343e654f1b33220220af76db914dc1d1
```

Comment:
I think that we actually want to plumb this through this action's inputs, but I am too dumb to understand the `EXTRA_ARG_` stuff.